lib/lita/adapters/slack.rb

```diff
@@ -9,7 +9,9 @@ class Slack < Adapter
       # Required configuration attributes.
       config :token, type: String, required: true
       config :proxy, type: String
-      config :parse, type: String
+      config :parse, type: [String, NilClass]
+      config :link_names, type: [FalseClass, TrueClass, NilClass]
```

Comment:
This is a clearer way to represent it:

``` ruby
config :link_names, type: [true, false], default: false
```

The same can be done for `unfurl_links`.